src/containers/Layout.tsx

```diff
@@ -17,19 +17,21 @@ interface LayoutProps {
 const Layout = ({ children, title }: LayoutProps) => {
   return (
     <>
-      <Helmet>
-        <title>{title ? `${title} | Leleka` : "Leleka"}</title>
-      </Helmet>
-      <GeneralContainer className="container-xxl">
-        <ErrorsToaster />
-        <div className="row">
-          <LeftPanel />
-          <main className="col-xl-6 col-lg-6 col-md-11 col-sm-11 col-12">
-            {children}
-          </main>
-          <RightPanel />
-        </div>
-      </GeneralContainer>
+      <HelmetProvider>
+        <Helmet>
+          <title>{title ? `${title} | Leleka` : "Leleka"}</title>
+        </Helmet>
+        <GeneralContainer className="container-xxl">
+          <ErrorsToaster />
+          <div className="row">
+            <LeftPanel />
+            <main className="col-xl-6 col-lg-6 col-md-11 col-sm-11 col-12">
+              {children}
+            </main>
+            <RightPanel />
+          </div>
+        </GeneralContainer>
+      </HelmetProvider>
```

Comment:
is it not possible to resolve error with normal react-helmet? could you add reference to github issue/stackoverflow where you found this solution

Reply:
https://github.com/nfl/react-helmet/issues/548